hub/models.py

```diff
@@ -37,14 +59,38 @@ def __str__(self):
         return self.name
 
 
-class EduSource(TimestampedModel):
+# A resource for learning a set of topics (e.g. a video or pdf)
+# Resources have prereqs (RequiredTopics) and CoveredTopics
+# Users can rate Resources based on how good they are and how technical they are
+class Resource(TimestampedModel):
+    VIDEO = 0
+    WIKI = 1
+    SLIDES = 2
+    LECTURE = 3 # Video leture
+    COURSE = 4
+    RESEARCH_PAPER = 5
+    PDF = 6
+    PROBLEM_SET = 7
+
+    TYPE_CHOICES = (
+        (VIDEO, "Video"),
+        (WIKI, "Wiki"),
+        (SLIDES, "Slides"),
+        (LECTURE, "Lecture"),
+        (COURSE, "Course"),
+        (RESEARCH_PAPER, "Research Paper"),
+        (PDF, "PDF"),
+        (PROBLEM_SET, "Problem Set"),
+    )
+
     name = models.CharField(max_length=500)
-    type = models.CharField(max_length=500)
+    type = models.CharField(max_length=30, choices=TYPE_CHOICES, default=VIDEO)
```

Comment:
So it seemed like doing a charfield with choices was better because now when we poll "Resource.type" from the DB it will be "Video" instead of 0, etc. though there's a bug with this code here that I'm working out before it's ready to merge